Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivity.kt

```diff
@@ -181,7 +181,8 @@ class MainActivity : AppCompatActivity(), HasAndroidInjector {
     private fun navigateByIntentUri(intent: Intent?) {
         val uri = intent?.data ?: return
         Timber.i("Uri:$uri")
-        navController.navigate(CheckInsFragment.createCheckInUri(uri.toString()))
+        val rootUri = uri.toString().replace(".app/?", ".app?")
```

Comment:
The actual uri does not include `/` in `.app/?`, but the system adds it when it is scanned from Google Lens.
The link still valid ,but it won't match any `regexDescriptors`  from `AppConfig` and aborts the CheckIn process 🙅🏼

Reply:
I am no regex expert, but I think it should be possible to modify our descriptors so that both versions match (with and without the `/`). Then we wouldn't need this...

Reply:
The descriptor comes from the config. @mlenkeit  can you update the regex to match both?

Reply:
Aligned with Max , Config should match both